### PR TITLE
Decision-making at Sourcegraph

### PR DESCRIPTION
**Pull request by dadlerj**

### CODE REVIEW

**Review by christinaforney — APPROVED** [review]

**Review by felixfbecker — APPROVED** [review]
This is explained really nicely and feels fitting for Sourcegraph (e.g. I like the appeal to not get hung up in overkill matrixes) 💯

File: handbook/communication/decisions.md [hunk 1]
```diff
@@ -1,23 +1,82 @@
...
+
+### Examples
+
+While the principle above is clear, how to apply it can be murky. Examples below are helpful to illustrate it.
+
+#### Changing our team’s OKR for this quarter
+
+- Because the team itself is responsible for the execution and results of their own OKRs, the [decider](#decider) should be clear: the team manager must make the decision to change an OKR.
+
+#### Choosing software to use for expense reimbursements
```
Comment (sqs) [inline handbook/communication/decisions.md:63]: Clarify that this is reimbursement software like Expensify not about choosing a tool for which you will be reimbursed

**Review by sqs — APPROVED** [review]
Looks great

File: handbook/communication/decisions.md [hunk 2]
```diff
@@ -1,23 +1,82 @@
...
+- This is a more complex situation, of course, as everyone at the company will _use_ this software. The [recommender](#recommenders) should collect input from a sample of teammates to verify the new process will be successful.
+
+#### Building a new Growth team to focus on user growth
+
+- While user growth is a company-wide goal, it primarily aligns with the Product and Marketing organizations (based on team-level OKRs and efforts). Other teams affected include Sales, Engineering, Business Operations, and more.
+- As a result, the Product and Marketing teams are the ones who are primarily responsible for the results of such a decision. In this case, greater impact will be felt on the Product and Engineering organizations versus Marketing, as they may have to reallocate teammates to this new team full-time.
+- Given the goal that would drive this decision is cross-functional, the heads of Marketing and Engineering should be consulted for input, or potentially even given [approver](#approvers) rights, but in this case, the [decider](#decider) is the head of the Product organization.
+
+Please provide examples where decision rights were clear and made a decision successful!
```
Comment (nicksnyder) [inline handbook/communication/decisions.md:74]: Is this sentence out of place? Probably should be below the "examples" heading?

**Review by nicksnyder — APPROVED** [review]

File: handbook/communication/decisions.md [hunk 3]
```diff
@@ -1,23 +1,82 @@
...
+
+Our high-agency culture is only successful if individuals across the organization are empowered to make decisions. If you or someone on your team is rarely making decisions, that’s a sign of organizational failure. Managers at Sourcegraph are celebrated for delegating decision-making by empowering their teams.
+
+### Examples
+
+While the principle above is clear, how to apply it can be murky. Examples below are helpful to illustrate it.
+
+#### Changing our team’s OKR for this quarter
+
+- Because the team itself is responsible for the execution and results of their own OKRs, the [decider](#decider) should be clear: the team manager must make the decision to change an OKR.
```
Comment (dan-mckean) [inline handbook/communication/decisions.md:61]: This gets a little messier for engineering teams when you have a PM and an EM.

Comment (nicksnyder) [reply handbook/communication/decisions.md:61]: EM and PM are a team and need to agree on OKRs. If they can't come to an agreement, then they do a clean escalation.

https://about.sourcegraph.com/handbook/communication/conflicts#clean-escalation

Comment (dan-mckean) [reply handbook/communication/decisions.md:61]: Niceeee! I love it when I discover new nuggets of useful wisdom in the handbook!

But the commented text suggests that in all teams, the decider of OKR changes is a singular manager. So maybe worth changing @dadlerj (perhaps by adding a caveat about engineering teams and the fact that decisions there are sometimes owned by both PM and EM).

Comment (dadlerj) [reply handbook/communication/decisions.md:61]: I added a note on this. I called out the risk here (I _do_ fear joint decision-making has risks, and can lead to [diffusion of responsibility](https://en.wikipedia.org/wiki/Diffusion_of_responsibility)) but happy to leave it in while it is what the Product and Eng teams use.

**Pull request merged**